### PR TITLE
Added Terms conditions

### PR DESCRIPTION
**Pull request by AswaniBolisetti**
#139 

@notsoocool 
I have added a page for Terms and Conditions successfully,
Could you please consider it as level2 if you like it and it took so much effort and I have completed all the issues raised by me

Kindly check it out 

https://github.com/user-attachments/assets/7c8e0660-b01f-4a7a-941a-53a450a7648d

### CODE REVIEW

**Comment by notsoocool** [comment]
Hi @AswaniBolisetti,

Kindly follow the sleek and minimalist design for your **TC** page and also for the footer. Please avoid raising a PR just for the sake of raising one and earning points.

Thanks!

**Comment by notsoocool** [comment]
Please have a diff PR for each issues.

**Comment by AswaniBolisetti** [comment]
> Please have a diff PR for each issues.

@notsoocool 
I have created different branches for each issue and raised different pull requests

I think you confused, actually it shows all the pull requests created by me in every issue that doesn't mean I created single pr
Kindly check it out

It took so much effort even I have my mid exam today, i completed this but you closed all the pull requests

**Comment by notsoocool** [comment]
> > Please have a diff PR for each issues.
> 
> @notsoocool I have created different branches for each issue and raised different pull requests
> 
> I think you confused, actually it shows all the pull requests created by me in every issue that doesn't mean I created single pr Kindly check it out
> 
> It took so much effort even I have my mid exam today, i completed this but you closed all the pull requests

Actually you are not getting it @AswaniBolisetti, see you have to create a different branch by fetching the develop branch, then you have to make the specific changes in the specified branch. What you did is you made all the changes in the same branch and raised 3 different PRs. As you can see in the "files changed," I can see all the changes that should have been done separately in different PRs, which makes it difficult for the Maintainer to review.

First of all, it's my fault that I assigned you 3 issues at a time, but since you have initiated 3 issues and got assigned to them, now try to solve them accordingly.
you initiated 3 issues and got assigned to it now try to solve it,

**Comment by AswaniBolisetti** [comment]
> > > Please have a diff PR for each issues.
> > 
> > 
> > @notsoocool I have created different branches for each issue and raised different pull requests
> > I think you confused, actually it shows all the pull requests created by me in every issue that doesn't mean I created single pr Kindly check it out
> > It took so much effort even I have my mid exam today, i completed this but you closed all the pull requests
> 
> Actually you are not getting it @AswaniBolisetti, see you have to create a different branch by fetching the develop branch, then you have to make the specific changes in the specified branch. What you did is you made all the changes in the same branch and raised 3 different PRs. As you can see in the "files changed," I can see all the changes that should have been done separately in different PRs, which makes it difficult for the Maintainer to review.
> 
> First of all, it's my fault that I assigned you 3 issues at a time, but since you have initiated 3 issues and got assigned to them, now try to solve them accordingly. you initiated 3 issues and got assigned to it now try to solve it,

@notsoocool 
No, I created three different branches(enhance-footer,terms-conditions, scroll-progress) and raised three different PRs

File: components/ui/scroll-progress.tsx
Comment (notsoocool) [inline components/ui/scroll-progress.tsx:1]: then why is this scrollprogress file in terms condition PR?????

File: components/global/footer.tsx
Comment (notsoocool) [inline components/global/footer.tsx:1]: Then why is this footer file in terms condition PR??????

**Comment by notsoocool** [comment]
> > > > Please have a diff PR for each issues.
> > > 
> > > 
> > > @notsoocool I have created different branches for each issue and raised different pull requests
> > > I think you confused, actually it shows all the pull requests created by me in every issue that doesn't mean I created single pr Kindly check it out
> > > It took so much effort even I have my mid exam today, i completed this but you closed all the pull requests
> > 
> > 
> > Actually you are not getting it @AswaniBolisetti, see you have to create a different branch by fetching the develop branch, then you have to make the specific changes in the specified branch. What you did is you made all the changes in the same branch and raised 3 different PRs. As you can see in the "files changed," I can see all the changes that should have been done separately in different PRs, which makes it difficult for the Maintainer to review.
> > First of all, it's my fault that I assigned you 3 issues at a time, but since you have initiated 3 issues and got assigned to them, now try to solve them accordingly. you initiated 3 issues and got assigned to it now try to solve it,
> 
> @notsoocool No, I created three different branches(enhance-footer,terms-conditions, scroll-progress) and raised three different PRs

I reviewed your file and now i get it, you actually created different branch from the same branch you made your commit, that is why its reflecting the other changes

**Comment by AswaniBolisetti** [comment]
@notsoocool 
Yes

**Pull request closed without merging**